src/plugins/vis_augmenter/public/types.ts

```diff
@@ -24,3 +68,14 @@ export interface VisLayerExpressionFn {
 }
 
 export interface AugmentVisSavedObject extends SavedObject, ISavedAugmentVis {}
+
+export interface VisLayerResponseValue {
+  visLayers: object;
+}
+
+export type VisLayerFunctionDefinition = ExpressionFunctionDefinition<
+  string,
+  VisLayerResponseValue,
+  any,
```

Comment:
`any` is for the arguments field, and it will need to stay loosely typed so plugins that utilize this function definition aren't restricted with how to pass arguments; we leave it up to the plugins to pass any arguments in any way they want.

Reply:
make sense, thanks!